src/probability_theory/stopping.lean

```diff
@@ -306,4 +309,154 @@ end linear_order
 
 end is_stopping_time
 
+section linear_order
+
+
+/-- Given a map `u : ι → α → E`, its stopped process with respect to the stopping
+time `τ` is the map `(i, x) ↦ u (min i (τ x)) x`.
+Intuitively, the stopped process stop evolving once the stopping time has occured. -/
```

Comment:
I guess it's easier to say "given a family of functions `u : ι → α → E`, the stopped process with respect to `τ` is `u i x` for `i < τ x`, and otherwise `u (τ x) x`." This docstring took me some reading